AtlasStarter/Program.cs

```diff
@@ -0,0 +1,185 @@
+
+            try
+            {
+                collection.InsertMany(docs);
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e);
+                return;
+            }
+
+            /*      *** FIND DOCUMENTS ***
+             * 
+             * Now that we have data in Atlas, we can read it. To retrieve all of
+             * the data in a collection, we call Find() with an empty filter. 
+             * The Builders class is very helpful when building complex 
+             * filters, and is used here to show its most basic use.
+             */
+
+            var allDocs = collection.Find(Builders<Recipe>.Filter.Empty)
+                .ToList();
+
+            foreach (Recipe doc in allDocs)
+            {
+                // Do something with each document ...
+            }
+
+            // We can also find a single document. Let's find the first document
+            // that has the string "potato" in the Ingredients list. Again we
+            // use the Builders class to create the filter, and a LINQ
+            // statement to define the property and value we're after:
+
+            var findFilter = Builders<Recipe>
+                .Filter.AnyEq(t => t.Ingredients,
+                "potato");
+
+            var findResult = collection.Find(findFilter).FirstOrDefault();
+
+            if (findResult == null)
+            {
+                return;
+            }
+
+            /*      *** UPDATE A DOCUMENT ***
+             * 
+             * You can update a single document, as shown here, or multiple 
+             * documents in a single call.
+             * 
+             * Here we update the PrepTimeInMinutes value on the document we 
+             * just found.
+             */
+
+            var updateFilter = Builders<Recipe>.Update.Set(t => t.PrepTimeInMinutes, 72);
+
+            // The follwing FindOneAndUpdateOptions specify that we want the *updated* document
+            // to be returned to us. By default, we get the document as it was *before*
+            // the update.
+
+            var options = new FindOneAndUpdateOptions<Recipe, Recipe>()
+            {
+                ReturnDocument = ReturnDocument.After
+            };
+
+            // The updatedDocument object is a Recipe object that refelcts the
+            // changes we just made.
+            var updatedDocument = collection.FindOneAndUpdate(findFilter,
+                updateFilter, options);
+
+            /*      *** DELETE DOCUMENTS ***
+             *      
+             *      As with other CRUD methods, you can delete a single document 
+             *      or all documents that match a specified filter. To delete all 
+             *      of the documents in a collection, pass an empty filter to 
+             *      the DeleteMany() method:
+             */
+
+            var deleteResult = collection
+                .DeleteMany(Builders<Recipe>.Filter.Empty);
```

Comment:
Wondering if it would be a good idea to leave behind some documents so users can see them in Atlas. Thoughts?

Reply:
I can go either way; I think the user can comment out the Delete line, or pause the execution, if they want to snoop around their Atlas collection. I think I'd prefer that over leaving orphaned data, but I'm open to what the group decides.

Reply:
100% waffling on this one, too.  On the one hand, I like the idea of tidying up after ourselves.  On the other, we're already creating and leaving behind a `testDatabase` and `testCollection`, so we could just leave behind a test doc, too.

Reply:
+1 for leaving behind a few sample docs and directing users to the data explorer to go check them out, delete them, or similar... great way to encourage more discovery of valuable product surface are.

Reply:
opted for leaving 2 docs behind, as per Jesse's comment, which is non-waffly.